.env.production

```diff
@@ -0,0 +1,33 @@
+## Auth Microservice ##
```

Comment:
I thought we were removing the .env.(enviroment) files

Reply:
See similar note and response on pull request of similar branch in orange api, here https://github.com/amida-tech/orange-api/pull/94#pullrequestreview-150221040